src/ApiCallJS.js

```diff
@@ -547,7 +552,13 @@ export async function createProfile(user) {
         EndPoints.users;
 
       await sendToEndPoint(config, 'POST', url, accessToken, user).then(
-        function (responseText) {
+        async function (responseText) {
+          var appId = JSON.parse(await EncryptedStorage.getItem(appIdC));
+          await EncryptedStorage.setItem(appId + user_auth, JSON.stringify({
```

Comment:
`verifyTwoFactor` will need similar logic. Access tokens/client ID will be returned **only if** user account exists